netplotbrain/plot.py

```diff
@@ -128,10 +128,15 @@ def plot(nodes=None, fig: Optional[plt.Figure] = None, ax=None, view: str = 'L',
     nrows, view, frames = _nrows_in_fig(view, frames)
     
     #if subtitles is not specified, give each subplot its view name
-    if subtitles is None:
+    if subtitles == 'auto':
         subtitles = []
         for i in range(0, nrows * frames):
             subtitles.append('auto')
+    #if subtitles is None, remove subtitles 
+    elif subtitles == None:
+        subtitles=[]
+        for i in range(0, nrows * frames):
+            subtitles.append('')
```

Comment:
`elif subtitles is None:` (not ==)

`subtitles = [''] * (nrows * frames)`

would be more efficient way of doing this to avoid the forloop

Reply:
also I think it would be better if this was `[None] * (nrows * frames)` instead of empty string